apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -1849,7 +1849,7 @@ export interface Extensions {
                 }
             };
         };
-        notificationChannel: {
+        emailAndSMS: {
```

Comment:
I think we can keep the old name here, for the following reasons.

- These are code level changes and does not affect the end UX
- notificationChannel is generic, therefore when we add new channels, we don't need to update the variables again.

WDYT?

Reply:
Notification channels is not a preferred name by the team for this component. Since this is a still new component, I think it is better if we change it now

Reply:
cc: @brionmario

Reply:
I'm also +1 to treat these as separate features.
UI grouping can be something secondary.